cli/azd/internal/repository/app_init.go

```diff
@@ -1107,12 +1113,73 @@ func promptSpringBootVersion(console input.Console, ctx context.Context) (string
 	}
 }
 
+func promptMissingEventHubsNameOrExit(console input.Console, ctx context.Context, eventHubs *appdetect.AzureDepEventHubs) {
+	switch eventHubs.MavenDependencyType {
+	case appdetect.SpringIntegrationEventHubs, appdetect.SpringMessagingEventHubs, appdetect.SpringKafka:
+		eventHubsNames, err := promptEventHubsNames(console, ctx)
+		if err != nil {
+			console.Message(ctx, fmt.Sprintf("Error happened when prompt eventhubs name: %s.", err))
+			os.Exit(-1)
+		}
+		for i, eventHubsName := range eventHubsNames {
+			propertyName := string(eventHubs.MavenDependencyType) + strconv.Itoa(i)
+			eventHubs.EventHubsNamePropertyMap[propertyName] = eventHubsName
+		}
+	case appdetect.SpringCloudStreamEventHubs, appdetect.SpringCloudStreamKafka:
+		promptMissingPropertyAndExit(console, ctx, "spring.cloud.stream.bindings.<binding name>.destination")
+		os.Exit(0)
+	case appdetect.SpringCloudEventHubsStarter:
+		promptMissingPropertyAndExit(console, ctx, "spring.cloud.azure.eventhubs.event-hub-name or "+
+			"spring.cloud.azure.eventhubs.[producer|consumer|processor].event-hub-name")
+		os.Exit(0)
+	}
+}
+
 func promptMissingPropertyAndExit(console input.Console, ctx context.Context, key string) {
 	console.Message(ctx, fmt.Sprintf("No value was provided for %s. Please update the configuration file "+
 		"(like application.properties or application.yaml) with a valid value.", key))
 	os.Exit(0)
 }
 
+func promptEventHubsNames(console input.Console, ctx context.Context) ([]string, error) {
```

Comment:
nit: add comment like this: `// todo: delete this after get event hub name from java codes.`

Reply:
Added.